tests/test_eload_backlog.py

```diff
@@ -151,7 +151,7 @@ def test_set_project_and_analysis(self):
                 [('ERZ9997',), ('ERZ9998',), ('ERZ9999',)]  # self.analysis_accessions
             ]
             assert eload.project_accession == 'PRJEB9999'
-            assert m_get_results.call_args[0][1] == ("select project_accession from evapro.project "
+            assert m_get_results.call_args[0][1] == ("select project_accession from evapro.sql.project "
```

Comment:
What's the reason for this change?

Reply:
That an automated refactor that I manually reverted. Apparently not everywhere...